### DISCUSSION TITLE
Why did you run out of manufacturers?

### DISCUSSION THREAD

**Discussion by PawAlldeller** [vosen/ZLUDA#95] [General]
First of all, thank you for taking the initiative to start this project!
I bow down! You fulfill your wishes =)

I saw a text in the description about non-interested parties.
Of course, AMD is falling before our eyes...I wrote about a similar implementation 3 years ago, but to no avail.
The project seems to be "Tower", aimed at this, but it's strangely slow development.

Therefore:
You have China...A competitor who is potentially interested in growth.
Although the capacity is far from the same, but I think it will be easier to agree?

**Comment by vosen** [comment]
I ask myself the same question :)
As for China, I was not contacted by anyone
